src/wallets/browser/NearWallet.ts

```diff
@@ -24,7 +24,8 @@ class NearWallet extends BrowserWallet implements INearWallet {
     const state = getState();
     if (!state.nearConnection) return;
     this.wallet = new WalletConnection(state.nearConnection, "near_app");
-    EventHandler.callEventHandler("init");
+    console.log("emit")
```

Comment:
I think we put this in to debug the `init` issue but in the end decided to remove the event entirely as there isn't a use case for it yet